modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -52,11 +55,16 @@ The following problems may exist when you have a server group composed of multip
 
 * *Salesforce streaming API*: if multiple instances of the same application are deployed, they will fail since the API only supports a single consumer. No failover support in case the instance connected is stopped or crashes
 
+== Prerequisites
 
+* You must have at least two Mule runtimes in a cluster, each of which should run on different physical (or virtual) machines.
+* Mule HA requires a reliable network connection between servers to maintain synchronization between the nodes in the cluster.
+* You must keep the following ports open to set up a Mule cluster: port `5701` and port `54327`.
+* You need to enable the multicast IP address: `224.2.2.3`. Multicast is used to discover new cluster members.
 
-== About Clustering
+== Clustering
 
-With out of the box configurations, it is recommended that you scale a cluster to no more than eight Mule runtimes. All the Mule runtimes in a cluster group together to form a single unit. Thus, you can deploy, monitor, or stop all the Mule runtimes in a cluster as if they were a single Mule runtime. If you need more nodes than eight, contact https://www.mulesoft.com/support-and-services/mule-esb-support-license-subscription[MuleSoft Support].
+With out of the box configurations, it is recommended that you scale a cluster to no more than eight Mule runtimes. All the Mule runtimes in a cluster group together to form a single unit. Thus, you can deploy, monitor, or stop all the Mule runtimes in a cluster as if they were a single Mule runtime.
```

Comment:
I think we're supposed to avoid "out of the box" and "out-of-the-box" (even though it's everywhere in our docs).  Maybe "default configurations"? I'm not sure what the best replacement is here.